jdbc/src/main/java/org/apache/zeppelin/jdbc/JDBCInterpreter.java

```diff
@@ -550,9 +549,9 @@ public Connection getConnection(String dbPrefix, InterpreterContext context)
         LOGGER.debug("createSecureConfiguration() returned");
         boolean isProxyEnabled = Boolean.parseBoolean(
                 getProperty("zeppelin.jdbc.auth.kerberos.proxy.enable", "true"));
-        if (basePropertiesMap.get(dbPrefix).containsKey("proxy.user.property")
+        if (basePropertiesMap.get(DEFAULT_KEY).containsKey("proxy.user.property")
```

Comment:
BTW, can we change `basePropertiesMap` not to use DEFAULT_KEY as a key?

Reply:
Ref.

https://github.com/apache/zeppelin/blob/master/jdbc/src/main/java/org/apache/zeppelin/jdbc/JDBCInterpreter.java#L216-L225

Reply:
> BTW, can we change `basePropertiesMap` not to use DEFAULT_KEY as a key?

Hi @jongyoul ,I totally agree with you.  
I plan to refactor `JDBC` module  in the very soon future.  

As I was developing, I realized that removing `basePropertiesMap` would require a significant code change. I don't want this PR to get too big , so I plan to refactor `JDBC` module in another PR.